src/integrationTest/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/component/testutils/WithHomeOfficeStatusSearchStub.java

```diff
@@ -0,0 +1,125 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.component.testutils;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.matchingJsonPath;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.matching.RequestPatternBuilder.newRequestPattern;
+import static uk.gov.hmcts.reform.iahomeofficeintegrationapi.component.testutils.IntegrationConstants.HOME_OFFICE_API_URL;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.util.ResourceUtils;
+
+public interface WithHomeOfficeStatusSearchStub {
+
+
+
+    default void addHomeOfficeApiSearchValidResponseStub(WireMockServer server, String homeOfficeRefefence)
```

Comment:
Maybe consider whether this really needs to be an interface if everything is a default method?  It's test code so it's ok but just for FYI.

Reply:
Agree. I kept it to have same pattern across our APIs.